src/components/connection_handler/src/connection_handler_impl.cc

```diff
@@ -297,8 +297,9 @@ uint32_t ConnectionHandlerImpl::OnSessionStartedCallback(
   if (!AllowProtection(get_settings(), service_type, is_protected)) {
     return 0;
   }
+  const uint32_t connection_key = KeyFromPair(connection_handle, session_id);
```

Comment:
@okozlovlux Just to let you know that [PR](https://github.com/smartdevicelink/sdl_core/pull/1433) has already contained a similar fix.